step-functions/step-functions-handler/src/main/java/step/functions/handler/AbstractFunctionHandler.java

```diff
@@ -269,18 +269,20 @@ protected Output<OUT> delegate(String functionHandlerClassname, Input<IN> input)
 		return delegate(ApplicationContextBuilder.MASTER, functionHandlerClassname, input);
 	}
 
-	private class FileVersionCloseable implements AutoCloseable {
+	private static class FileVersionCloseable implements AutoCloseable {

```

Comment:
Exactly the same comment as for the other class, guarantee non-null in constructor, omit null check in close.